src/gen-ts.ts

```diff
@@ -552,6 +556,27 @@ function handleProperties(analyzerProperties: Iterable<analyzer.Property>):
   return tsProperties;
 }
 
+function generateObjectMap(feature: analyzer.Element): ts.Property[] {
```

Comment:
Would prefer this returned `ts.Property|undefined` and check for undefined in the places above, instead of returning an array here. A little more verbose, but clearer I think.